bin/overrides.js

```diff
@@ -19,15 +18,13 @@ module.exports = {
   hset: {
     overwrite: false,
     defs: [
-      "$1(key: RedisKey, object: Record<string, string | Buffer | number>, callback?: Callback<number>): Result<number, Context>",
-      "$1(key: RedisKey, map: Map<string | Buffer | number, string | Buffer | number>, callback?: Callback<number>): Result<number, Context>",
+      "$1(key: RedisKey, object: object, callback?: Callback<number>): Result<number, Context>",
```

Comment:
Should we keep the map declaration to have better hints?
How is the non-string key support for the type `object`?
```js
      "$1(key: RedisKey, object: object, callback?: Callback<number>): Result<number, Context>",
      "$1(key: RedisKey, map: Map<string | Buffer | number, string | Buffer | number>, callback?: Callback<number>): Result<number, Context>",
```

Reply:
`Map` is also `object` but yeah let's keep them for documentation purposes. Only utf8 keys are supported for `object` so users have to use `Map` if they need others.